### DISCUSSION TITLE
关于在控件绑定事件，出现事件冒泡以及传递EventArgs 的问题

### DISCUSSION THREAD

**Discussion by fvffv** [BYJRK/DotNet-Discussions#239] [WPF 开发]
如题 
我在实际开发中遇到了这样一个问题
![微信截图_20240203211832](https://github.com/BYJRK/DotNet-Discussions/assets/46892450/3295a866-7253-44a4-853b-af97ac020452)
一个border里面套了一个grid
两个控件都各自设置了事件触发
当我点击grid的时候会先触发父控件的按下事件，导致无法触发子控件的松开事件，我查阅资料知道EventArgs 里可以设置Handled，但我却没办法获取到EventArgs 

主要就是不想让他出现事件冒泡这种情况

**Comment by BYJRK** [comment]
这不是冒泡或者隧道的问题。你的两个控件捕获的是完全不同的两种事件，那么当你的鼠标按下或者释放时，就是会被控件捕获响应的事件。这两个事件互相之间没有路由关系。

**Reply by fvffv** [reply]
可现在他有个问题我不知道如何处理，当我在子控件左键按下的时候就会被父控件捕获，导致接下来的子控件左键放开事件无法触发了
